solutions/usaco-988.mdx

```diff
@@ -112,4 +115,72 @@ print(ans, file=open('photo.out', 'w'))
 ```
 
 </PySection>
+<JavaSection>
+	
+**Time Complexity:** $\mathcal{O}(N^2)$
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class Photoshoot {
+	public static void main(String[] args) throws IOException {
+		Kattio io = new Kattio("photo");
+
+		int N = io.nextInt();
+		int[] bessieList = new int[N];
+		int[] correctList = new int[N];
+		// Used to prevent repeated values in our permutation
+		boolean[] isItThere = new boolean[N];
+
+		for (int i = 1; i < N; i++) {
+			bessieList[i] = io.nextInt();
+		}
+
+		// Iterate through every possible starting cow and see if we can
+		// construct a permutation based on that.
+		for (int i = 1; i <= N; i++) {
+			correctList[0] = i;
+			isItThere = new boolean[N];
+			for (int j = 1; j < N; j++) {
+				correctList[j] = bessieList[j] - correctList[j - 1];
+			}
+			boolean bad = false;
+			for (int k : correctList) {
+				// Our permutation isn't valid because there are unacceptable values
+				// So, we can stop searching. 
+				if (k > N || k <= 0) {
+					bad = true;
+					break;
+				}
+				else {
+					// Our permutation isn't valid because some elements repeat
+					if (isItThere[k - 1]) {
+						bad = true;
+						break;
+					}
+					else {
```

Comment:
```suggestion
					} else {
```
can you do this for the other if/else's as well
the current brace style is a blight upon this earth